inscriptions/ISic003548.xml

```diff
@@ -190,35 +195,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI337212">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#GC">
                 <ab>
-                    <lb n="1"/>
-
-
-                    θεοῖς πᾶσι<lb n="2"/>Δρεπαναί
-                    <supplied reason="lost">ων</supplied>
+                    <lb n="1"/>θεοῖς πᾶσι
+                    <lb n="2"/>Δρεπαναί<supplied reason="lost">ων</supplied>
                     <lb n="3"/>πόλεω<supplied reason="lost">ς</supplied> σ<supplied reason="omitted">ω</supplied>τῆρσιν
-                    <lb n="4"/>
-                    <supplied reason="lost">τ</supplied>ραπέ<supplied reason="lost">ζ</supplied>α<supplied reason="omitted">ν
-                </supplied> καὶ τ<supplied reason="omitted">ὸ</supplied>ν βωμὸν
-                    <lb n="5"/>
-                    <expan>
-                        <abbr>Λ</abbr>
-                        <ex>ούκιος</ex>
-                    </expan>
-                    Μάλιος
-                    <supplied reason="lost">— —</supplied>
-                    <lb n="6"/>χι<supplied reason="omitted">λ</supplied>ιάρχησα
-                    <supplied reason="lost">ς</supplied>
-                    <lb n="7"/>Π Π <surplus>²⁷p(ropria) p(ecunia) or p(ecunia) p(osuit) (?) (Tybout)</surplus>²⁷.
-                    <lb n="8"/>
+                    <lb n="4"/><supplied reason="lost">τ</supplied>ραπέ<supplied reason="lost">ζ</supplied>α<supplied reason="omitted">ν</supplied> καὶ τ<supplied reason="omitted">ὸ</supplied>ν βωμὸν
+                    <lb n="5"/><expan><abbr>Λ</abbr><ex>ούκιος</ex></expan> Μάλιος<supplied reason="lost">— —</supplied>
```

Comment:
`<supplied reason="lost">— —</supplied>`
should become:
`<gap reason="lost" extent="unknown" unit="character"/>`